File: Python/leetcode_657.py
def judge_circle(moves: str) -> bool:
    horizont = 0
    vertical = 0
    for move in moves:
        if move == "U":
            vertical += 1
        elif move == "D":
            vertical -= 1
        elif move == "L":
            horizont -= 1
        else:
            horizont += 1
    return horizont == 0 and vertical == 0

File: JavaScript/leetcode_657.js
/**
 * @param {string} moves
 * @return {boolean}
 */
var judgeCircle = function(moves) {
    let h = 0;
    let v =0;
    for (let index = 0; index < moves.length; index++) {
        switch (moves[index]) {
            case "U":
                h++
                break;
            case "D":
                h--
                break;
            case "L":
                v--
                break;
            default:
                v++
                break;
        }; 
    };
    return v === 0 && h === 0
};